@Test public void testPow() throws Exception { assertEquals(Fraction.ONE, Fraction.ONE.pow(0)); assertEquals(Fraction.ONE, Fraction.ONE.pow(1)); assertEquals(Fraction.ONE, Fraction.ONE.pow(2)); assertEquals(Fraction.ONE, Fraction.ONE.pow(3)); assertEquals(Fraction.ONE, Fraction.ONE.pow(4)); assertEquals(Fraction.ONE, Fraction.ONE.pow(5)); assertEquals(Fraction.ONE, Fraction.ONE.pow(6)); assertEquals(Fraction.ONE, Fraction.ONE.pow(7)); }